libqtile/widget/bluetooth.py

```diff
@@ -32,7 +32,9 @@
 
 class Bluetooth(base._TextBox):
     """
-    Displays bluetooth status or connected device.
+    Displays bluetooth status for a particular connected device.
+
+    (For example your bluetooth headphones.)
 
     Uses dbus to communicate with the system bus.
```

Comment:
Can you also fix this to say it uses `dbus-next` (rather than `dbus`).

Reply:
Yeah